### ISSUE TITLE
Integration with Publons

### ISSUE THREAD

**Issue by robintw** [openjournals/joss-reviews#48]
Publons (see https://publons.com/) is an online service that helps you get credit for peer reviewing work. A number of my colleagues have started using it to keep track of which journals they have reviewed for, and to allow them to easily show their contribution to 'the academic community' in job applications, funding applications etc.

It'd be great if JOSS could integrate with Publons? (I believe they have an API that can be used fairly easily - and they've been very responsive to me with questions I've asked previously).

What do people think?

**Comment by kyleniemeyer** [comment]
Yeah, I think something like this would be nice. Based on my use of Publons so far, would this require sending out an email to each reviewer to acknowledge their work?

**Comment by arfon** [comment]
> What do people think?

@robintw - I think this is a great idea. @arhpreston can Publons pick up GitHub issues as reviews or would we need to push data to Publons _from_ JOSS?

**Comment by cMadan** [comment]
I think Publons is great and it would be great to integrate our review process with them.

**Comment by robintw** [comment]
Looking at their site, it seems like there is an API you can use (see https://publons.com/api/v2/ and specifically https://publons.com/api/v2/review/post/), but you have to be a 'partner' first (see https://publons.com/partner/info/). The alternative is to do it via emails as @kyleniemeyer said.

Hopefully with the API there could be some sort of automatic integration put together?

**Comment by benmarwick** [comment]
I see increasing use of Publons among my colleagues, so I think it would be good if JOSS claimed a formal presence there. It might help to raise the status of JOSS as a serious journal, which is important for some reviewers.

Seems like the only way to initiate this is by making contact by email partnerships@publons.com (and maybe paying them?).

Currently we see two possible matches:

- https://publons.com/journal/38378/the-journal-of-open-source-software    
- https://publons.com/journal/38377/journal-of-open-source-software     

it's not clear which is actually JOSS, so that's a bit confusion, and bad for JOSS' image. Perhaps if the editorial board had profile on the site and 'claimed' one of these it would be more obvious which one that reviewers should associate with.

**Comment by arhpreston** [comment]
Hi there. You've spotted a duplicate caused by listing the journal in CrossRef under two names. We've unified the two records here:
https://publons.com/journal/38378/the-journal-of-open-source-software

Hope that helps!

Please do drop us a line on partnerships@publons.com if you want to discuss further.

**Comment by andrewheiss** [comment]
Has there been any progress on this? Do e-mails get sent out after reviews are completed, or does Publons accept GitHub issues as proof?

Thanks!
